### PR TITLE
hsetroot: update to 1.0.5.

### PR DESCRIPTION
**Pull request by fnogcps**

### CODE REVIEW

File: srcpkgs/hsetroot/template [hunk 1]
```diff
@@ -1,18 +1,24 @@
 # Template file for 'hsetroot'
 pkgname=hsetroot
-version=1.0.2
-revision=9
-wrksrc="hsetroot-${version}.orig"
-build_style=gnu-configure
+version=1.0.5
+revision=10
```
Comment (Hoshpak) [inline srcpkgs/hsetroot/template]: revision should be reset to 1 on a version bump

File: srcpkgs/hsetroot/template [hunk 2]
```diff
@@ -1,18 +1,24 @@
...
+wrksrc="hsetroot-${version}"
+build_style=gnu-makefile
 hostmakedepends="automake libtool pkg-config"
-makedepends="libX11-devel imlib2-devel"
+makedepends="libX11-devel imlib2-devel libXinerama-devel"
+depends="imlib2"
```
Comment (Hoshpak) [inline srcpkgs/hsetroot/template]: This is automatically detected, it does not ned to be added explicitely:

```
SONAME: libImlib2.so.1 <-> imlib2>=1.4.2_1
```

File: srcpkgs/hsetroot/template [hunk 3]
```diff
@@ -1,18 +1,24 @@
...
-homepage="https://directory.fsf.org/wiki/Hsetroot"
-distfiles="${DEBIAN_SITE}/main/h/hsetroot/hsetroot_${version}.orig.tar.gz"
-checksum=597fe1175e590d54d0927095ce361c86ddef15ebc70a96c4bf509f6cb9165723
+homepage="https://github.com/himdel/hsetroot"
+distfiles="https://github.com/himdel/hsetroot/archive/${version}.tar.gz"
+checksum=cff5f83b85a21e0c5c8c8eecbd552090bb75d8ac59e27b0bae48046f9cb5a44a
 
-pre_configure() {
-	autoreconf -if
+do_build() {
```
Comment (Hoshpak) [inline srcpkgs/hsetroot/template]: The build_style does cover that, no need to override the function.

File: srcpkgs/hsetroot/template [hunk 4]
```diff
@@ -1,18 +1,24 @@
...
+	make
+}
+
+do_install() {
```
Comment (Hoshpak) [inline srcpkgs/hsetroot/template]: This should also be left to the build style, the makefile has an install target.

**Comment by fnogcps** [comment]
Tried to rebuild the binary package, but I got an error.

https://gist.github.com/fnogcps/a3db1349a0e2591f8daa14dcde8c4ffa

**Comment by fnogcps** [comment]
I made a mistake in my commit, sorry for that, I created a new pull request. 

#19164

**Pull request closed without merging**